CMakeLists.txt

```diff
@@ -185,12 +185,18 @@ if (NOT ELOGIND_FOUND AND NOT SYSTEMD_FOUND)
     set(REBOOT_COMMAND "/sbin/shutdown -r now")
 endif()
 
+if (SYSTEMD_FOUND)
+    set(RUNTIME_DIR_DEFAULT "/run")
+else()
+    set(RUNTIME_DIR_DEFAULT "${CMAKE_INSTALL_FULL_LOCALSTATEDIR}/run")
+endif()
+
 
 # Set constants
 set(DATA_INSTALL_DIR            "${CMAKE_INSTALL_FULL_DATADIR}/sddm"                CACHE PATH      "System application data install directory")
 set(DBUS_CONFIG_DIR             "${CMAKE_INSTALL_SYSCONFDIR}/dbus-1/system.d"       CACHE PATH      "DBus config files directory")
 set(STATE_DIR                   "${CMAKE_INSTALL_FULL_LOCALSTATEDIR}/lib/sddm"      CACHE PATH      "State directory")
-set(RUNTIME_DIR                 "${CMAKE_INSTALL_FULL_LOCALSTATEDIR}/run/sddm"      CACHE PATH      "Runtime data storage directory")
+set(RUNTIME_DIR                 "${RUNTIME_DIR_DEFAULT}/sddm"                       CACHE PATH      "Runtime data storage directory")
```

Comment:
Could you move the `/sddm` part into `RUNTIME_DIR_DEFAULT` as well so that `RUNTIME_DIR_DEFAULT` is actually the default of `RUNTIME_DIR`?